middleware/logger.js

```diff
@@ -0,0 +1,56 @@
+const shortId = require('shortid');
```

Comment:
This file has a ton of console logs, mostly for good reason, so just to shut up ESLint for this file add an ignore at the top:
```suggestion
/* eslint-disable no-console */
const shortId = require('shortid');
```